src/TestStack.BDDfy.Tests/Scanner/FluentScanner/StepTitleTests.cs

```diff
@@ -29,6 +31,58 @@ public void MethodCallInStepTitle()
             story.Scenarios.Single().Steps.ElementAt(6).Title.ShouldBe("And with foo arg");
         }
 
+        [Fact]
+        public void TitleFunctionCanBeOverriden()
+        {
+            FooClass something = new FooClass();
+            var context = TestContext.GetContext(something);
+            context.FluentScanner = new TestStack.BDDfy.FluentScanner<FooClass>(something);
+            context.FluentScanner.SetCreateTitle((a, b, c, d, e) => new StepTitle("hallo"));
+            var story = something
+               .Given(_ => GivenWeMutateSomeState())
```

Comment:
I think it would be more in keeping with the FluentScanner design to implement the SetCreateTitle functionality as extension methods, in the same way as the Given, When, Then methods are implemented. That way the FluentScanner constructor can remain internal and not be made public.